components/playlist/browser/resources/components/playlistItem.tsx

```diff
@@ -156,11 +167,17 @@ export default function PlaylistItem ({
 
   return (
     <PlaylistItemContainer
-      onClick={() => !showingMenu && onClick(id)}
+      onClick={() => !showingMenu && onClick(item)}
       onMouseEnter={() => setHovered(true)}
       onMouseLeave={e => setHovered(false)}
     >
       <a ref={anchorElem} href={`#${id}`} />
+      {isEditing &&
```

Comment:
I don't know if it'll work for you, but there's also a checkbox component in Leo
https://leo.bravesoftware.com/?path=/story/checkbox--primary

Should be something like:

```tsx
import Checkbox from '@brave/leo/react/checkbox'

<Checkbox checked={isSelected} onChange={() => onClick(item)}/>
```

might not be quite what you need though :smile:

Reply:
Thanks, didn't know about it. It seems the checkbox is a rectangular one, but I need a circular one here, so it'd be better to leave this as is 😄